src/OpenLoco/src/Vehicles/OrderManager.cpp

```diff
@@ -502,10 +502,10 @@ namespace OpenLoco::Vehicles::OrderManager
                     // Find the vehicle that has the order
                     for (auto* head : VehicleManager::VehicleList())
                     {
-                        if (head->orderTableOffset >= i
+                        if (i >= head->orderTableOffset
```

Comment:
This looks like [Yoda notation](https://en.wikipedia.org/wiki/Yoda_conditions) to me. Assuming this is correct, I'd prefer to use `head->orderTableOffset < i` instead.